src/commands/aksDeleteCluster/aksDeleteCluster.ts

```diff
@@ -33,6 +35,14 @@ export default async function aksDeleteCluster(_context: IActionContext, target:
 
         if (succeeded(result)) {
             vscode.window.showInformationMessage(result.result);
+
+            // Periodically refresh the subscription treeview, because the list-clusters API
+            // call still includes the cluster for a while after it's been deleted.
+            refreshIntervals.forEach((interval) => {
+                setTimeout(() => {
+                    vscode.commands.executeCommand("aks.refreshSubscription", clusterNode.result.subscriptionTreeNode);
+                }, interval * 1000);
```

Comment:
do we really need this periodic refresh instead user can refresh himself right ? just thinking,,